lib/rubocop/cop/lint/percent_string_array.rb

```diff
@@ -55,7 +55,7 @@ def contains_quotes_or_commas?(node)
 
         def autocorrect(node)
           lambda do |corrector|
-            node.values.each do |value|
+            node.each_child_node do |value|
               range = value.loc.expression
 
               match = range.source.match(TRAILING_QUOTE)
```

Comment:
```Ruby
# Original lib/rubocop/cop/lint/percent_string_array.rb
def autocorrect(node)
  lambda do |corrector|
    node.values.each do |value|
      ...
```

Calling `#values` on `ArrayNode` simply calls `#each_child_node.to_a`. It is pointless to call `#to_a` if you are just going to call `#each` on it after, therefore, just use `#each_child_node`

```Ruby
# Modified lib/rubocop/cop/lint/percent_string_array.rb
def autocorrect(node)
  lambda do |corrector|
    node.each_child_node do |value|
      ...
```